source/compatibility.tex

```diff
@@ -786,8 +786,8 @@
 semantics in this revision of \Cpp{}. Implementations may choose to
 translate trigraphs as specified in \CppXIV{} if they appear outside of a raw
 string literal, as part of the \impldef{mapping from physical source file characters
-to basic source character set} mapping from physical source file characters to
-the basic source character set.
+to translation character set} mapping from physical source file characters to
+the translation character set.
```

Comment:
The paper asked to keep "basic" here; should this change be in a separate commit?

Reply:
It's not really fixed.